components/org.wso2.carbon.appmgt.sample.deployer/src/main/java/org/wso2/carbon/appmgt/sample/deployer/deploy/Deployer.java

```diff
@@ -128,4 +136,18 @@ public boolean deploySample(NativeObject request) throws AppManagementException
         }
         return response;
     }
+
+    private boolean isSubscriptionEnabled() {
+        AppManagerConfiguration appManagerConfiguration = ServiceReferenceHolder.getInstance()
+                .getAPIManagerConfigurationService().getAPIManagerConfiguration();
+        Boolean isSubscriptionEnabled = false;
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/765a403f533c64e20704c6b0424d16f7acee08d0